docs/source/evaluation.ipynb

```diff
@@ -9810,7 +9810,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from weatherbench2.config import Selection, Paths, DataConfig, EvalConfig"
+    "from weatherbench2.config import Selection, Paths, Data, Eval"
```

Comment:
Can we use `config.Selection`, etc, for the examples here?

Reply:
Yes, will change now.